ale_linters/python/mypy.vim

```diff
@@ -37,8 +37,9 @@ function! ale_linters#python#mypy#GetCommand(buffer) abort
     " we can know with certainty the base path for the 'filename' keys below.
     return ale#path#CdString(l:dir)
     \   . ale#Escape(l:executable) . l:exec_args
-    \   . ' --show-column-numbers '
+    \   . ' '
     \   . ale#Var(a:buffer, 'python_mypy_options')
```

Comment:
```suggestion
    \   . ale#Pad(ale#Var(a:buffer, 'python_mypy_options'))
```

Use `ale#Pad` instead of adding a space before the option. This makes tests easier to maintain.